crashreports/internal/win32/CrashReportContext.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <string>
+#include <map>
+
+class CrashReportContext
+{
+	static constexpr int MaxBufferLength{ 2048 };
+	static constexpr int MaxCommandLength{ 8192 };
+
+	wchar_t senderPath[MaxBufferLength]{};
```

Comment:
Audacity has a convention (not always very strictly followed) that non-static member variables have names beginning with "m" followed by a capital.  (And "s" for static).  Exceptions are for POD (plain-old-data) struct classes that don't need to maintain important invariant conditions, but this isn't a class like that.

Yes, "m_" might be a better convention.  I wouldn't mind encouraging that for new code.

Reply:
@Paul-Licameli `m` or `m_` usual using for coloring private class members, and `_` in end of name (google style). Better if we using `s` or `s_`. for example: `static int sSomStatic` or `static int s_SomeStatic`, I think second case better looking with `s_`.